tests/test_ods_backend.py

```diff
@@ -123,12 +123,22 @@ def test_simple(rmock):
                             '?format=shp&timezone=Europe/Berlin'
                             '&use_labels_for_header=true')
 
+    resource = test_b.resources[4]
+    assert resource.title == 'gtfs.zip'
+    assert resource.description == 'GTFS 15/01'
+    assert resource.format is None
```

Comment:
Format should be `zip`, no ? (Until we have better support for zip internals)

Reply:
Done. I tried something with the `mimetypes` lib instead of simply taking `mimetype.split('/')[-1]`, don't know if it's the right solution.